contracts/ownership/Ownable.sol

```diff
@@ -35,6 +38,7 @@ contract Ownable {
   function transferOwnership(address newOwner) onlyOwner {
     require(newOwner != address(0));      
     owner = newOwner;
+    OwnershipTransferred(owner, newOwner);
```

Comment:
This is failing to log the previous owner because it's updated in the previous line. The event should be emitted first, and then the state variable updated.